core/src/main/resources/reference.conf

```diff
@@ -245,6 +245,11 @@ engine {
   }
 }
 
+languages {
```

Comment:
Placeholder to remind me i was going to talk to you in person, assuming my thought doesn't change after i go through the whole PR